kolibri/plugins/coach/assets/src/state/getters/reportUtils.js

```diff
@@ -0,0 +1,69 @@
+
+const ReportConstants = require('../../reportConstants');
+const CoreConstants = require('kolibri.coreVue.vuex.constants');
+
+const ContentNodeKinds = CoreConstants.ContentNodeKinds;
+
+
```

Comment:
agree.

Note that most of these functions pre-dated the current refactor, and I'm considering removing some...